src/main/webapp/script.js

```diff
@@ -17,12 +17,43 @@ $('#myModal').on('shown.bs.modal', function () {
 });
 
 function initializeModal(pathId, itemId, itemName) {
-    console.log("initializing", arguments)
+    console.log("initializing", arguments);
+
+    //var myvariable = "${pathId}";
+    //alert("PathId is " + myvariable);
 
     $("#feedback-title").text(itemName);
+    $("#feedback-id").text(itemId);
     const form = $("#feedback");
 }
 
 function submitFeedback() {
-  // TODO
-}
+    console.log("Sending the submission form info.");
+    const itemId = document.getElementById("feedback-id").innerText;
+    const pathId = document.getElementById("path-id").innerText;
+    const options = document.getElementById("ratingValues");
+    const rating = options.options[options.selectedIndex].value;
+
+    const XHR = new XMLHttpRequest();
```

Comment:
Is there a reason you chose to go with XHR over Fetch?

Reply:
Not really. I can change it.

Reply:
If it's not too much to do yea that'd be cool,  fetch is the more recent solution.

Reply:
Should the rating also be passed in the url?

Reply:
I think if have it set to look in the request body for the rating and the completed flag like so 
```
   boolean completed = Boolean.parseBoolean(request.getParameter("completed"));
   int rating = Integer.parseInt(request.getParameter("rating"));
```

So you should probably go with the body, it should look similar to this (what we did for week 2)

```
    const body = new URLSearchParams();
    body.append('text', text);

    const res = await fetch("ENDPOINT_URL", {
        method: 'POST',
        body
    });
```

Reply:
but with the proper values (rating, completed) of course.

Reply:
Just remember to wrap it all in an async function if you want to use the await keyword.